### PR TITLE
NSFS | NC | CLI | Remove unwanted console logs for manage NSFS script

### PR DESCRIPTION
**Pull request by naveenpaul1**
### Explain the changes
Remove unwanted console logs for manage NSFS script

### Issues: Fixed #xxx / Gap #xxx
1. creation_date is missing bucket
2. CLI argument validation issue for account and bucket
3. unit test for managing CLI 
4. error and success messages updated
5. CLI validation issue fixed
6. health nsfs service name change to noobaa_nsfs

Note: _log printing to stderr changes are not included._

### Testing Instructions:
1. run both account and bucket operations

- [ ] Doc added/updated
- [X] Tests added

### CODE REVIEW

File: src/server/object_services/schemas/nsfs_bucket_schema.js [hunk 1]
```diff
@@ -17,6 +17,12 @@ module.exports = {
         name: {
             type: 'string',
         },
+        system_owner: {
+            type: 'string',
```
[the remark this answers is not in the document]
Comment (naveenpaul1) [reply src/server/object_services/schemas/nsfs_bucket_schema.js:21]: @romayalon I added to validate the type of this system_owner and bucket_owner, Now we are not checking the type in CLI validation.

File: docs/non_containerized_NSFS.md [hunk 2]
```diff
@@ -395,31 +395,35 @@ These error codes will get attached with a specific Bucket or Account schema ins
 Users can create, update, delete, and list buckets and accounts using CLI. If the config directory is missing CLI will create one and also create accounts and buckets sub-directories in it and default config directory is `${config.NSFS_NC_DEFAULT_CONF_DIR}`. 
 
 CLI will never create or delete a bucket directory for the user if a bucket directory is missing CLI will return with error.
+
+
```
Comment (romayalon) [inline docs/non_containerized_NSFS.md:399]: can you please also change the NSFS_SERVICE_FAILED on health error codes to the new error code please?

File: src/cmd/manage_nsfs.js [hunk 3]
```diff
@@ -622,84 +648,91 @@ async function get_view_config_data(config_file_path) {
...
 
+function is_undefined(value) {
+    if (value === 'undefined') return true;
+    return false;
+}
+
+
 exports.main = main;
-if (require.main === module) main();
+if (require.main === module) main();
```
Comment (romayalon) [inline src/cmd/manage_nsfs.js:738]: missing new line at the end here and in config.js

**Review by romayalon — APPROVED** [review]
2 lasts comments and let's merge :)

**Pull request merged**